lib/attributor/flatpack/config.rb

```diff
@@ -1,10 +1,22 @@
 module Attributor
   module Flatpack
     class Config < Attributor::Hash
+      DEFAULT_SEPARATOR = '_'.freeze
       @key_type = Symbol
 
+      class << self
+        def separator(sep = nil)
+          return @separator unless sep
+
+          @separator = sep
+        end
+      end
+
       def self.inherited(klass)
         super
+        klass.instance_eval do
+          @separator = Attributor::Flatpack::Config::DEFAULT_SEPARATOR
```

Comment:
Seems this might lead to an odd case where inheritance "reverts" this being set, but I can't really see a reason to subclass this, change the separator, and then inherit again.